core-src/annotatemain.cc

```diff
@@ -634,8 +637,10 @@ static bool move_and_record(int attacker)
             } else {
                 const bool success = move.scan(moveline.c_str());
                 if (!success) {
-                    puts("The given string did not parse as a move. It was:");
-                    printf("\"%s\"\n", moveline.c_str());
+                    if (strlen(moveline.c_str()) > 0) {
+                        puts("The given string did not parse as a move. It was:");
+                        printf("\"%s\"\n", moveline.c_str());
+                    }
```

Comment:
The point of this is to be less spammy when the user hits "return" without entering a move? Okay, pulled into master as  4536b7f813f2f1749f3bc4fbd7cb43e981fa0033.